app/src/main/java/com/jetbrains/handson/mpp/mobile/RecyclerViewAdapter.kt

```diff
@@ -0,0 +1,35 @@
+package com.jetbrains.handson.mpp.mobile
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+
+class RecyclerViewAdapter(private val mList: List<RecyclerViewCell>) : RecyclerView.Adapter<RecyclerViewAdapter.ViewHolder>() {
```

Comment:
Can we use a more descriptive name than mList please

Reply:
Perhaps also for the adapter and view holder, the name is quite generic when this file is currently specific to e.g. a recycler which shows departureTime and arrivalTime. Perhaps this should be the DepartureBoardRecyclerViewAdapter